app/src/main/java/eu/kanade/tachiyomi/data/backup/restore/BackupRestorer.kt

```diff
@@ -157,10 +166,27 @@ class BackupRestorer(
         )
     }
 
+    private fun CoroutineScope.restoreExtensionRepos(backupExtensionRepo: List<BackupExtensionRepos>) = launch {
+        ensureActive()
+        try {
+            extensionRepoRestorer(backupExtensionRepo)
+        } catch (e: Exception) {
+            errors.add(Date() to "${e.message}")
+        }
```

Comment:
this will fail the whole thing if any repo restore fails. Should have a onError lambda to pass the info here instead of throwing inside the other function.